htdocs/modules/system/themes/modern/tests/ThemeCompatibilityTest.php

```
@@ -0,0 +1,443 @@
+            $this->fail();
+        }
+    }
+
+    /**
+     * Test 8: Dependencies
+     */
+    private function testDependencies()
+    {
+        echo "→ Testing external dependencies... ";
+
+        $depsOk = true;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$depsOk'. (undefined)

(UnusedLocalVariable)